fossevents/templates/base.html

```diff
@@ -16,14 +16,39 @@
     {% compress css %}
     <link rel="stylesheet" type="text/css" href="{% static 'css/normalize.css' %}" />
     <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
+      <link type="text/css" href="{% static 'css/style.css' %}" rel="stylesheet">
```

Comment:
You already have `main.scss` let's make use of it. Instead of writing styles in pure css.